app/src/common/shared/org/mozilla/vrbrowser/browser/engine/Session.java

```diff
@@ -328,20 +331,20 @@ private void restore() {
         dumpAllState();
     }
 
-    private SessionState createSession() {
+    private SessionState createSession(boolean aOpen) {
         SessionSettings settings = new SessionSettings.Builder()
                 .withDefaultSettings(mContext)
                 .build();
 
-        return createSession(settings);
+        return createSession(settings, aOpen);
     }
 
-    private SessionState createSession(@NonNull SessionSettings aSettings) {
+    private SessionState createSession(@NonNull SessionSettings aSettings, boolean aOpen) {
         SessionState state = new SessionState();
         state.mSettings = aSettings;
         state.mSession = createGeckoSession(aSettings);
 
```

Comment:
Please use enum instead of a boolean so it is more obvious what is happening in the code.